packages/babel-plugin-remove-undefined-if-possible/src/index.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+function removeRvalIfUndefined(declaratorPath) {
+  const rval = declaratorPath.get("init")
+                              .evaluate();
```

Comment:
can you push evaluate to the same line. It's not really long to be added to a separate line.